lab3.md

```diff
@@ -324,4 +326,66 @@ In the example we used `sort` from [BusyBox](https://busybox.net/downloads/BusyB
 
 > Tip: did you know that OpenFaaS supports Windows binaries too? Like C#, VB or PowerShell?
 
+### Bonus: Combine all your functions in one yml file
+
+Creating multiple functions means having a `.yml` file for each function.
```

Comment:
There are two concepts here:

* stack.yml as a default (not needing a flag)

* combining two functions into on YAML file

@BurtonR  Please could you reword to explain in separate sections / points rather than combined?

I also merged another PR so this needs a rebase:

```
git fetch origin
git rebase origin/master
```